helpers/url.js

```diff
@@ -0,0 +1,6 @@
+import { Platform } from 'react-native';
+
+export const baseUrl = Platform.OS === 'android' ? 'http://10.0.2.2' : 'http://127.0.0.1';
```

Comment:
usually, folks developing on ios and android would only search for the ip address excluding the protocol, etc

Also, in general its a good practice to reduce duplication.

Reply:
are you suggesting to have the protocol in a different variable?

Reply:
yes.